lib/parsers/mpeg_parser.rb

```diff
@@ -0,0 +1,115 @@
+
+# MPEG Headers documentation:
+# http://dvd.sourceforge.net/dvdinfo/mpeghdrs.html#seq
+# http://www.cs.columbia.edu/~delbert/docs/Dueck%20--%20MPEG-2%20Video%20Transcoding.pdf
+# Useful tool to check the file information: https://www.metadata2go.com/
+class FormatParser::MPEGParser
+  include FormatParser::IOUtils
+
+  MPEG_VALUES = {
+    'mpg' => :mpg,
+    'mpeg' => :mpeg
+  }
+
+  ASPECT_RATIOS = {
+    1 => '1:1',
```

Comment:
This one is tricky but usually aspects in, for instance, DPX headers are represented as a pair of shorts. What other options can we use for this value that would not have the caller perform string parsing?

Reply:
I didn't understand this :/. Can you explain, please?

The aspect ratio has the following format `number : number`, so I think and string is the best way to store that value

Reply:
No problem - aspect ratios are sometimes represented as a pair of `short`s (8-bit unsigned integers) which the caller can the convert to a representation it finds convenient. One system might want to do the calculations in integers, another system might have actual decimal arithmetic available and can prefer that. For example, the DPX format will specify the 1.33333:1 aspect ratio (standard for video / 35mm film) as a tuple of `[4, 3]` which the caller then can divide using arithmetic it finds more convenient. `1.6666666...:1` then becomes `[16, 9]` etc. For aspects which do not have a close integer equivalent you can use `[235, 100]`. Alternatively maybe we can introduce some "minmum viable" FormatParser-wide decimal type?

Reply:
Perfect, I understand now. All the examples I saw online had a `decimal:decimal` format, that's why I was confused. As an MVP version, should we leave this as it is?

Reply:
As an MVP - ok 👍